tests/integration_tests/build/test_dependencies.py

```diff
@@ -47,23 +47,17 @@ def test_num_dependencies(dep_file):
     # appear multiple times).
     for line in deps:
         if line and "(*)" not in line:
-            current_deps.add(line)
+            # only care about dependency name, not version/path/github repo
+            current_deps.add(line.split()[0])
 
     # Use the code below to update the expected dependencies.
-    # from pprint import pprint
     # with open(dep_file, "w", encoding='utf-8') as prev_deps:
-    #      pprint(sorted(current_deps), stream=prev_deps)
+    #     prev_deps.write(repr(sorted(current_deps)).replace(',', ',\n'))
 
     with open(dep_file, encoding='utf-8') as prev_deps:
         prev_deps = ast.literal_eval(prev_deps.read())
-    if len(current_deps) > len(prev_deps):
-        difference = current_deps - set(prev_deps)
-        msg = "The number of build dependencies has increased." \
-              " Is this expected? New dependencies {}".\
-            format(list(difference))
-        assert False, msg
-    elif len(current_deps) != len(prev_deps):
-        msg = "The build dependencies have changed." \
-              " Use the code above to modify the {} file.". \
-            format(dep_file)
-        assert False, msg
+
+    difference = current_deps - set(prev_deps)
+
+    if difference:
+        assert False, f"New build dependencies detected. Is this expected? New dependencies {difference}"
```

Comment:
This line looks longer than 80, how come no linter complained?

Reply:
The warning was disabled in this commit.
https://github.com/firecracker-microvm/firecracker/pull/3262/commits/f8f21d36f3ffc30e3ee971741664231c3f9fee20

Reply:
I think that might be related to this: https://github.com/firecracker-microvm/firecracker/pull/3292

Reply:
black chose some... interesting formatting for that line, lol